mbed/mbed.py

```diff
@@ -3011,6 +3011,7 @@ def dev_mgmt(toolchain=None, target=None, source=False, profile=False, build=Fal
     dict(name=['-i', '--ide'], help='IDE to create project files for. Example: UVISION4, UVISION5, GCC_ARM, IAR, COIDE'),
     dict(name=['-m', '--target'], help='Export for target MCU. Example: K64F, NUCLEO_F401RE, NRF51822...'),
     dict(name='--source', action='append', help='Source directory. Default: . (current dir)'),
+    dict(name=['--profile'], action='append', help='Path of a build profile configuration file. Default: mbed-os/tools/profiles/debug.json'),
```

Comment:
I would like the default to say "debug". That's the option that users are likely to use.

Reply:
By "default" do you mean add that as part of the args? So like:
```suggestion
    dict(name=['--profile'], default='debug', action='append', help='Path of a build profile configuration file. Default: mbed-os/tools/profiles/debug.json'),
```

Reply:
Nope. I was thinking in the text like:

```suggestion
    dict(name=['--profile'], action='append', help='Path of a build profile configuration file. Default: debug'),
```

Reply:
Ahh I see, would you like me to update the other `--profile` help texts in the other commands?